test/integration/query-interface.test.js

```diff
@@ -41,6 +41,28 @@ describe(Support.getTestDialectTeaser('QueryInterface'), () => {
     });
   });
 
+  describe('showAllTables', () => {
+    it('should contain only tables', function() {
+      const cleanup = r => {
+        return this.sequelize.query('DROP VIEW V_Fail').then(() => r, () => r);
+      };
```

Comment:
Wouldn't using `DROP VIEW IF EXISTS` simplify this catch-and-ignore pattern here? If yes, please change this to:

```js
const cleanup = () => this.sequelize.query('DROP VIEW IF EXISTS V_Fail');
```

If not, please tell me why and regardless refactor this to:

```js
const cleanup = () => this.sequelize.query('DROP VIEW V_Fail').reflect();
```

Which is more readable.

Reply:
The `IF EXISTS` only became supported in MS SQL 2016. So I'm a bit afraid that adding `IF EXISTS` would break tests in some old databases. Unless the CI pipeline has the oldest versions of each RDBMS that is officially supported.

Reply:
Makes sense. Please use my second suggestion then, and add a comment above it explaining what you just said.